Classes/Eel/MailObfuscator.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Networkteam\Neos\MailObfuscator\Eel;
+
+use Neos\Eel\ProtectedContextAwareInterface;
+use Networkteam\Neos\MailObfuscator\Converter\Mailto2HrefObfuscatingConverter;
+use Networkteam\Neos\MailObfuscator\Converter\RewriteAtCharConverter;
+
+/**
+ * Eel helpers to provide MailObfuscator functions
+ */
+class MailObfuscator extends RewriteAtCharConverter implements ProtectedContextAwareInterface {
+
```

Comment:
I would prefer to not extend `\Networkteam\Neos\MailObfuscator\Converter\RewriteAtCharConverter` but injecting the converter objects via interface.

```php
class MailObfuscator implements ProtectedContextAwareInterface {

    /**
     * @var EmailLinkNameConverterInterface
     * @Flow\Inject
     */
    protected $emailLinkNameConverter;

    /**
     * @var MailtoLinkConverterInterface
     * @Flow\Inject
     */
    protected $mailtoLinkConverter;

   ...

}
```